### DISCUSSION TITLE
Unix socket support

### DISCUSSION THREAD

**Discussion by lbensaad** [pocketbase/pocketbase#3233] [Q&A]
I am using nginx as a reverse proxy, can pocketbase be started using unix socket for nginx to connect to it directly without the burden of tcp/ip?

**Comment by ganigeorgiev** [comment]
No, this is not supported at the moment.
Out of the box we allow listening only to TCP network address(es).

**Reply by vladimiroff** [reply]
Would you accept a PR that adds `--sock` persistent flag to the serve command, which if not empty listens on that socket, instead of running TCP listener?

**Reply by ganigeorgiev** [reply]
No, I don't think it would be a good idea as it could cause priority ambiguity with the existing `--http` and `--https` flags (which are already kindof confusing and we tried to simplify their usage in the past https://github.com/pocketbase/pocketbase/discussions/3190).

With that said, I've added a todo in the roadmap to at least consider eventually allowing custom listener creation from the `OnServe` hook but it remains a low priority for now.
